cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -61,6 +62,28 @@ func runProfile(cmd *cobra.Command, args []string) {
 		}
 	}
 	fmt.Printf("Profile '%s' set as active profile\n", profileName)
+
+	err := removeOcContext(lastActiveProfile)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+}
+
+func removeOcContext(profileName string) error {
+	if profileName == profileActions.GetActiveProfile() {
+		return nil
+	}
```

Comment:
I do not think we need to do this. Without this user can run the same `profile set` command twice and it will redo the every step. Sometimes it is useful when because of some reason things are out of sync.

Reply:
well that was the behaviour that i wanted to avoid, because it would unset the context for the active profile, well it will get reset again when the user starts minishift with `minishift start` but if the vm is already running and the user does profile set the context will be unset. and `oc` commands will not be able to run.

Reply:
IMO we should remove the check as it might help user to sync the context in case it is out of sync for the same active profile.